examples/pytorch/ogb/ogbn-mag/hetero_rgcn.py

```diff
@@ -29,13 +30,16 @@ def prepare_data(args):
 
     # train sampler
     sampler = dgl.dataloading.MultiLayerNeighborSampler([25, 20])
+    num_workers = 0 if device != "cpu" else int(os.cpu_count() / 2)
```

Comment:
I will suggest adding an argument to `ArgumentParser` and setting the default value to 0.